Wasted/FoodPlaceRatingSystem/RatingPopup.xaml.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using Rg.Plugins.Popup.Services;
+using Wasted.DummyAPI.BusinessObjects;
+using Wasted.FoodPlaceRatingSystem;
+
+namespace Wasted
+{
+    public partial class RatingPopup : Rg.Plugins.Popup.Pages.PopupPage
+    {
+
+        public String FoodPlaceTitle { get; }
+        public int FoodPlaceID { get; }
+        public int Rating
+        {
+            get
+            {
+                return ratingBar.SelectedStarValue;
+            }
+        }
```

Comment:
Readonly properties look nicer if implemented in this way:
public int Rating => ratingBar.SelectedStarValue;